### PR TITLE
Pz/types 003 - Categories endpoint

### PR DESCRIPTION
**Pull request by p-zielinski**
---
'@voucherify/sdk': minor
---
**Added support for following endpoints:**
- campaigns
  - POST /campaigns/campaignId/enable
    - method: `client.campaigns.enable(campaignId)`
  - POST /campaigns/campaignId/disable
    - method: `client.campaigns.disable(campaignId)`
- categories
  - GET, POST /categories
    - methods:
      - `client.categories.list()`
      - `client.categories.create(createCategory)`
  - GET, PUT, DELETE /categories/id
    - methods:
      - `client.categories.get(categoryId)`
      - `client.categories.update(categoryId, updateCategory)`
      - `client.categories.delete(categoryId)`

### CODE REVIEW

File: .changeset/comics-voucherify-two.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+ ---
+'@voucherify/sdk': minor
+---
+
+To run tests you need to copy packages/sdk/.env.example to packages/sdk/.env and fill the file with your credentials.
```
Comment (marcin-slezak) [inline .changeset/comics-voucherify-two.md:5]: Duplicated update about .env for a tests

**Review by marcin-slezak — CHANGES_REQUESTED** [review]
Why did we start from branch `types-002`? It could be just minor to the current major version.

File: packages/sdk/README.md [hunk 2]
```diff
@@ -372,6 +386,52 @@ client.distributions.publications.list(params)
...
 
+### Categories
+
+Methods are provided within `client.categories.*` namespace.
+
+- [Create Category](#create-category)
+- [Update Category](#update-category)
+- [Get Category](#get-category)
+- [Delete Category](#delete-category)
+- [Add Voucher to Campaign](#add-voucher-to-campaign)
```
[the remark this answers is not in the document]
Comment (p-zielinski) [reply packages/sdk/README.md:397]: Screenshot 2023-09-06 at 17 07 26

This is a list of `Categories API`
https://docs.voucherify.io/reference/category-object

File: .changeset/comics-voucherify-two.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ ---
+'@voucherify/sdk': minor
+---
+Added support for few missing endpoints:
+  - Added support for new endpoints: `/campaigns/campaignId/enable`,`/campaigns/campaignId/disable`, `/categories` and `/categories/id` [(examples of usage available in readme.md)](..%2F..%2Fpackages%2Fsdk%2FREADME.md)
```
Comment (marcin-slezak) [inline .changeset/comics-voucherify-two.md:5]: When you list using URI, you need to provide an HTTP method to be unequivocal

**Review by marcin-slezak — CHANGES_REQUESTED** [review]

**Review by marcin-slezak — CHANGES_REQUESTED** [review]
maybe some tests?

File: packages/sdk/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (p-zielinski) [reply packages/sdk/README.md]: 🤦 you are right

**Pull request closed without merging**